t/MaxMind/DB/Writer/Tree-record-collisions.t

```diff
@@ -551,9 +551,116 @@ use Net::Works::Network;
         'force_overwrite overwrites record even when merge_record_collisions is enabled',
         { merge_record_collisions => 1 },
     );
-}
+};
+
+subtest 'merge subrecord only if parent exists - hashes' => sub {
```

Comment:
It is. :-) But I can do a hash of a hash of a hash, although it shouldn't really matter given that the algorithm is recursive.